src/applications/ezr/config/chapters/householdInformation/spouseInformation.js

```diff
@@ -75,6 +85,13 @@ const spousalInformationPages = arrayBuilderPages(options, pageBuilder => ({
     uiSchema: spousePersonalInformationPageSchema.uiSchema,
     schema: spousePersonalInformationPageSchema.schema,
   }),
+  spouseAdditionalInformationPage: pageBuilder.itemPage({
+    title: content['household-spouse-addtl-info-title'],
+    path:
+      'household-information/spouse-information/:index/spouse-additional-information',
```

Comment:
iirc: @longmd had a comment about the path being slightly redundant and not needing `spouse` in the `/spouse-additional-information` part of the path. Not sure if you want to take his advice here